layers/meta-balena-jetson/recipes-bsp/tegra-binaries/tegra-flash-dry_36.3.0.bb

```diff
@@ -14,19 +14,26 @@ UEFI_CAPSULE:jetson-orin-nano-devkit-nvme = "TEGRA_BL_Orin_Nano.Cap.gz"
 UEFI_CAPSULE:jetson-orin-nx-seeed-j4012 = "TEGRA_BL_Seeed_j4012.Cap.gz"
 UEFI_CAPSULE:jetson-orin-nano-seeed-j3010 = "TEGRA_BL_Seeed_j3010.Cap.gz"
 UEFI_CAPSULE:jetson-agx-orin-devkit-64gb = "TEGRA_BL_3701_64.Cap.gz"
+UEFI_CAPSULE:forecr-dsb-ornx-orin-nano-8gb = "TEGRA_BL_Forecr_ORNX_Nano_8gb.Cap.gz"
```

Comment:
@asuykov could you please describe which exact changes have been applied for each of the boards when generating the boot blobs and the UEFI capsules? I.e what exact changes have been done to which files in the Nvidia BSP

Reply:
Done.